### PR TITLE
Allign MQTT config panel controls

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change
Improve the lay out for the MQTT config entry page.
Make topic field scale with the page and have a minimal width. Allign QoS fields, retain and subscribe button.

Large screen (from 600px width):
afbeelding

Narrow screen (275px):
afbeelding

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Put these fields in a `div` with `display: flex; justify-content: space-between;` give only the select a width, set `flex: 1;` on the textfield.

File: src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts [hunk 1]
```diff
@@ -47,32 +47,35 @@ class MqttSubscribeCard extends LitElement {
...
-          <mwc-button
-            .disabled=${this._topic === ""}
-            @click=${this._handleSubmit}
-            type="submit"
-          >
-            ${this._subscribed
-              ? this.hass.localize("ui.panel.config.mqtt.stop_listening")
-              : this.hass.localize("ui.panel.config.mqtt.start_listening")}
-          </mwc-button>
+          <div class="panel-dev-mqtt-subsribe-fields">
```
Comment (bramkragten) [inline src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts:50]:
```suggestion
          <div class="panel-dev-mqtt-subscribe-fields">
```

Comment (bramkragten) [reply src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts:50]: No need to be so specific with the class names btw, it is a shadow root, so isolated :-)

File: src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts [hunk 2]
```diff
@@ -170,6 +173,35 @@ class MqttSubscribeCard extends LitElement {
       pre {
         font-family: var(--code-font-family, monospace);
       }
+      @media screen and (min-width: 275px) {
+        .panel-dev-mqtt-subsribe-fields {
+          display: float;
```
[the remark this answers is not in the document]
Comment (jbouwh) [reply src/panels/config/integrations/integration-panels/mqtt/mqtt-subscribe-card.ts]: Thnx for helping out

**Review by bramkragten — APPROVED** [review]

**Pull request merged**